Makefile

```diff
@@ -0,0 +1,32 @@
+
+SHELL := /bin/bash
+
+install:
+	@echo Starting LunarVim Installer
+	bash ./utils/installer/install.sh
+
+uninstall:
+	@echo TODO: this is currently not supported 
+
+install-neovim-binary:
+	@echo Installing Neovim from github releases
+	bash ./utils/installer/install-neovim-from-release
+
+test:
+	bash ./utils/bin/test_runner.sh "$(UNIT)"
+
+lint: lint-lua lint-sh
+
+style: style-lua style-sh
+
+lint-lua:
+	luacheck *.lua lua/* tests/*
+lint-sh:
+	shfmt -f . | grep -v jdtls | xargs shellcheck
+
+style-lua:
```

Comment:
I updated both `shfmt` and `luacheck` to use `make` since they're a bit manual, and switched to a pre-made one for `stylua` which already has all the rules in a file independent of `make` anyway.